entry/magento-os/package.json

```diff
@@ -11,12 +11,14 @@
         "webpack-cli": "^5.1.4"
     },
     "scripts": {
-        "build-js": "webpack",
+        "build-js": "webpack --config webpack.config.dev.js",
```

Comment:
won't it work by just having argument "--dev"? withtou separate config file

Reply:
build-js probably need to stay to use it in normal mode, new one needed which will be used by watch